Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -2711,6 +2711,22 @@ TextStream& operator<<(TextStream& ts, const WebProcessProxy& process)
     appendIf(process.isUnderMemoryPressure(), "under-memory-pressure"_s);
     ts << ", " << process.throttler();
 
+#if PLATFORM(COCOA)
+    auto description = [](ProcessThrottleState state) -> ASCIILiteral {
+        switch (state) {
+        case ProcessThrottleState::Foreground: return "foreground"_s;
+        case ProcessThrottleState::Background: return "background"_s;
+        case ProcessThrottleState::Suspended: return "suspended"_s;
+        }
+        return "unknown"_s;
+    };
+
+    if (auto taskInfo = process.taskInfo()) {
+        ts << ", state: " << description(taskInfo->state);
+        ts << ", phys_footprint_mb: " << (taskInfo->physicalFootprint / 1048576.0) << " MB";
```

Comment:
Please use `MB` from StdLibExtras.h:

`(taskInfo->physicalFootprint / static_cast<double>(MB))` or `(taskInfo->physicalFootprint / MB)` if we don't need floating point precision.